README.rst

```diff
@@ -56,6 +57,8 @@ in search mode is provided below:
         bind_dn: "cn=hacker,ou=svcaccts,dc=example,dc=com"
         bind_password: "ch33kym0nk3y"
         #filter: "(objectClass=posixAccount)"
+        # Alternatively, simple binding with a custom bind pattern:
```

Comment:
Could you add some more comments to clarify this: an alternative to what? which attributes does this replace?